example/server/pushAzure.js

```diff
@@ -0,0 +1,52 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
```

Comment:
I checked with the MS owners of the code; they told me to remove the copyright since I had modified the file (I've removed it in an update). Apparently, the copyright info only applies to the unmodified version (which I did not use).